ci-operator/step-registry/baremetal/lab/test/cbo-extension/baremetal-lab-test-cbo-extension-commands.sh

```
@@ -0,0 +1,42 @@
+oc version --client
+oc wait nodes --all --for=condition=Ready=true --timeout=15m
+oc wait clusteroperators --all --for=condition=Progressing=false --timeout=15m
+
+TEST_SUITE="${TEST_SUITE:-cluster-baremetal-operator/firmware}"
+
+echo "Running CBO test suite: ${TEST_SUITE}"
+
+ret_value=0
+${BINARY} run-suite -c 1 --junit-path "${ARTIFACT_DIR}/junit.xml" "${TEST_SUITE}"
+
+echo "$(date +%s)" > "${SHARED_DIR}/TEST_TIME_TEST_END"
```

Shellcheck (0.11.0)

[style] Useless echo? Instead of 'echo $(cmd)', just use 'cmd'.

(SC2005)